configure

```diff
@@ -784,6 +784,7 @@ infodir
 docdir
 oldincludedir
 includedir
+runstatedir
```

Comment:
This is from the `autoconf` that Ubuntu ships. I don't really see the harm in having the `--runstatedir` option around.